src/OrleansServiceBus/Providers/Streams/EventHub/EventHubBatchContainer.cs

```diff
@@ -104,7 +104,17 @@ public bool ShouldDeliver(IStreamIdentity stream, object filterData, StreamFilte
             return true;
         }
 
-        internal static EventData ToEventData<T>(SerializationManager serializationManager, Guid streamGuid, String streamNamespace, IEnumerable<T> events, Dictionary<string, object> requestContext)
+        /// <summary>
+        /// Put events list and its context into a EventData object
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="serializationManager"></param>
+        /// <param name="streamGuid"></param>
+        /// <param name="streamNamespace"></param>
+        /// <param name="events"></param>
+        /// <param name="requestContext"></param>
+        /// <returns></returns>
+        public static EventData ToEventData<T>(SerializationManager serializationManager, Guid streamGuid, String streamNamespace, IEnumerable<T> events, Dictionary<string, object> requestContext)
```

Comment:
need to make this public so DataGenerator can use it to put the correct content into the EventData